justfile

```diff
@@ -17,6 +17,21 @@ lint:
 format:
```

Comment:
In commit message: "attempts to cove"

Reply:
I'm gonna go ahead and merge despite this. It could break `git log -S` but I don't know that anybody would ever search for the word "cover".